src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+    mark-section "Launching subjobs to run GATK GermlineCNVCaller"
+
+    export -f _upload_single_file  # required to be accessible to xargs sub shell
+
+    # Upload input files to workspace container so subjobs can access them
+    tsv=$( dx upload --brief /home/dnanexus/inputs/beds/annotated_intervals.tsv )
+
+    SECONDS=0
+    echo "Uploading ploidy and basecounts for sub jobs"
+
+    total_files=$(find /home/dnanexus/inputs/ploidy_dir /home/dnanexus/inputs/basecounts -type f | wc -l)
+    find /home/dnanexus/inputs/ploidy_dir /home/dnanexus/inputs/basecounts -type f \
```

Shellcheck

[warning] Use -print0/-0 or -exec + to allow for non-alphanumeric filenames.

(SC2038)